src/renderers/WebGLRenderer.js

```diff
@@ -2476,7 +2476,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 		var updateBuffers = false,
 			wireframeBit = material.wireframe ? 1 : 0,
-			geometryHash = ( geometry.id * 0xffffff ) + ( program.id * 2 ) + wireframeBit;
+			geometryHash = ( geometry.id * 0x1000000 ) + ( program.id * 4 ) + 2 + wireframeBit;
```

Comment:
Wait. So the main change here is that `BufferGeometry` is 2 units higher?
Doesn't seem like this is really solving the collision issue?